requirements.txt

```diff
@@ -3,7 +3,7 @@ numpy
 tabulate
 safetensors
 --pre
-tf-nightly==2.17.0.dev20240529
+tf-nightly>=2.17.0.dev20240529
```

Comment:
is the version number needed or can it be `tf-nightly` or perhaps `tf-nightly?=2.17`?

Reply:
We make changes in ai-edge-torch that depends on corresponding changes in tflite converter e.g. lowering of  composite, so `>=` is needed.